treeherder/etl/job_loader.py

```diff
@@ -131,7 +133,7 @@ def transform(self, pulse_job):
                 "log_references": self._get_log_references(pulse_job),
                 "artifacts": self._get_artifacts(pulse_job, job_guid),
             },
-            "coalesced": pulse_job.get("coalesced", []),
+            "superseded": pulse_job.get("superseded", []),
```

Comment:
I think this line needs to remain unchanged, since it's not referring to the job result, but this Pulse schema field:

```yaml
   coalesced:
     description: The job guids that were coalesced to this job.
     title: "coalesced"
     type: "array"
```

...and we can't change that field without bumping the Pulse schema version. (And in fact, I'd suggest when it comes time to bumping the schema, we just remove that field entirely.)

Reply:
Ahh, nice catch.  It's true that it should look from "coalesced" from the pulse object, but still store it in "superseded" because that's what ``store_job_data`` will be looking at.  Though it won't matter much since this is never sent in, afaik.  :)